frontend/src/components/AddRecipe.tsx

```diff
@@ -145,46 +150,51 @@ export default function AddRecipe(props: AddRecipeProps) {
             <Link to="/">
                 <img src="/icons8-zurück-48.png" alt="Back" />
             </Link>
-            <div className="recipe-card">
+            <div className="div_description">
+                <div className="recipe-title">
                 <h1>Neues Rezept</h1>
+                </div>
                 <form>
                     <h2>
                         <input
                             type="text"
-                            placeholder="Title"
+                            placeholder="Wie heißt dein Rezept?"
                             value={newRecipe.title}
-                            onChange={(e) => handleRecipeChange('title', e.target.value)}
+                            onChange={(event) => handleRecipeChange('title', event.target.value)}
+                            style={{ cursor: "pointer",width: "60em", height: "1.5em", marginBottom: "1em" }}
```

Comment:
Inline Style ist in der Berufswelt unerwünscht. Besser ist es einen `className` zu vergeben.

Reply:
same here... :)